openshift/hac-infra
# [HAC-1639] Updates to enable running hac-infra locally and in HAC Core

#### To test that `hac-infra` runs with `hac-core`:
- Pull the latest code from the HAC Core repo and run `yarn install`. Start hac-core by running `ENVIRONMENT=prod yarn dev` within the `frontend/` directory.
- Pull the changes from this PR, run `npm install` followed by `npm run start:federated`.
- Open the URL mentioned in the terminal output for hac-core: The `Workspaces` option should be visible in the left navigation menu. Clicking on it should show the empty Infra UI:
![image](https://user-images.githubusercontent.com/43621546/177837924-3480d220-3432-44e5-bc9c-41f37415d18c.png)

#### To test that `hac-infra` runs standalone:
```
npm run start:beta
```
Open https://stage.foo.redhat.com:1337/beta/hac in the browser.
